Python 3.14, edt v3.0.0, numpy 2.x, add maintainer

stuarteberg opened the pull request:
Attempting to simultaneously tackle #34 and #35.

- edt v3.0.0
- python 3.14
- numpy 2.0 (allow cython 3)
    - Related: [#31](https://github.com/conda-forge/edt-feedstock/pull/31) is now obsolete since the numpy migration [completed a while ago][1]
- Update license filepath/info
- Add myself as maintainer

[1]: https://conda-forge.org/news/2025/05/28/numpy-2-migration-closure/

---

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] ~Bumped the build number (if the version is unchanged)~
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

regro-cf-autotick-bot committed:
updated v3.0.0

regro-cf-autotick-bot committed:
Rebuild for python 3.14

regro-cf-autotick-bot committed:
MNT: Re-rendered with conda-smithy 3.52.0 and conda-forge-pinning 2025.09.01.08.01.4

xhochy committed:
Add setuptools to recipe meta.yaml

Add setuptools to the requirements.

stuarteberg committed:
add maintainer: stuarteberg

stuarteberg committed:
Update license info

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

stuarteberg committed:
Allow newer cython

stuarteberg committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.11.25.16.21.43

Other tools:
- conda-build 25.11.0
- rattler-build 0.42.1
- rattler-build-conda-compat 1.4.6

stuarteberg commented:
@william-silversmith This is passing!  See summary above.

william-silversmith commented:
Sounds good to me! Go ahead and add yourself as maintainer. I am cooking a new version of edt so there will be a new release pretty soon most likely.

stuarteberg commented:
Already added myself. :-)

Ready for merge!

stuarteberg commented:
Thanks, @william-silversmith !

Can you also have a look at this other feedstock:

https://github.com/conda-forge/connected-components-3d-feedstock/pull/35

maybe someday i can just write “conda install william-silversmith”
